hogvm/python/stl/date.py

```diff
@@ -0,0 +1,34 @@
+import datetime
+from typing import Optional
+
+import pytz
+
+
+def now():
+    return datetime.datetime.now()
```

Comment:
A little bit confused here - Typescript land has a custom object whereas python land has a native datetime? Is there not a risk here that the python version could behave quite differently?